ziti/contexts.go

```diff
@@ -0,0 +1,70 @@
+package ziti
+
+import (
+	"github.com/michaelquigley/pfxlog"
+	"github.com/openziti/sdk-golang/ziti/config"
+	"github.com/openziti/sdk-golang/ziti/edge"
+	"os"
+	"path/filepath"
+	"strings"
+	"sync"
+)
+
+var contexts = new(sync.Map)
+
+func init() {
+	ids := os.Getenv("ZITI_IDENTITIES")
+
+	idlist := strings.Split(ids, ";")
+
+	for _, id := range idlist {
+
+		if id == "" {
+			continue
+		}
+
+		_, err := LoadContext(id)
+		if err != nil {
+			pfxlog.Logger().Errorf("failed to load config from '%s'", id)
+			continue
+		}
+	}
+}
+
+func LoadContext(config_ string) (Context, error) {
```

Comment:
seems like this should be put next to `NewContextWithConfig` (ziti.go) and should also get a "WithOpts" example. Overall, I still don't think it belongs here, although I recognize it's convenient. it seems more like 'example code' to me.